packages/components/src/internal/components/assay/RunDataPanel.tsx

```diff
@@ -231,11 +255,29 @@ export class RunDataPanel extends PureComponent<Props, State> {
             wizardModel,
             getIsDirty,
             setIsDirty,
+            onResultsFileChange,
+            onResultsFileRemoval,
         } = this.props;
         const { message, messageStyle, previousRunData } = this.state;
         const isLoading = !wizardModel.isInit || queryModel.isLoading;
         const isLoadingPreview = previousRunData && !previousRunData.isLoaded;
         const columnMetadata = this.getEditableGridColumnMetadata();
+        const fileColumnNames = isLIMSEnabled()
+            ? wizardModel.assayDef.domainFileColumnNames(AssayDomainTypes.RESULT)
+            : undefined;
+        const assayFileFieldTooltip = (
```

Comment:
nit: Would be great if `<LabelOverlay/>` accepted `children` as the `content` so this could just be done inline. A bit of wasted processing going on here.

Reply:
yeah, not sure why `content` was added as a prop instead of just using `children`. I made that change, looks like there wasn't many usages.

Reply:
Cool. Thanks for doing that. 

nit: For this usage in particular the `assayFileFieldTooltip` could now just be inlined and would not be processed when there are no file columns.